src/firmware/boards/robot_stm32h7/BUILD

```diff
@@ -8,6 +8,10 @@ cc_binary(
         "freertos.c",
         "main.c",
     ],
+    copts = [
+        # We don't want to mark warnings as errors for 3rd party code
+        "-Wno-error",
+    ],
```

Comment:
Why do we need this now but not in the past?

Reply:
nuked! used https://gcc.gnu.org/onlinedocs/gcc-4.8.5/gcc/Diagnostic-Pragmas.html around the hal header